manifests/experiences/facebook/database.js

```diff
@@ -35,7 +37,9 @@ export default async function databaseBuilder(fileManager) {
   /// Advertisers who purchased your contact ////////////////////////////////////////////////////////////////////////////////
   db.create('advertisersContactInformation', [['name', 'TEXT']])
   const advertisersContactInformationFile = JSON.parse(
```

Comment:
Ok so that's an example where my suggestion does lead to uglier code, sorry

Maybe something like this looks slightly better ? Not sure it works though...

```
const advertisersContactInformationFileTexts = await fileManager.getPreprocessedTextFromId('advertisers-contact-list')
const advertisersContactInformationFile = advertisersContactInformationTexts.map(JSON.parse)[0]
```